website/project/model.py

```diff
@@ -730,6 +731,23 @@ def category_display(self):
         """The human-readable representation of this node's category."""
         return self.CATEGORY_MAP[self.category]
 
+    # We need the following 2 properties in order to serialize related links in NodeRegistrationSerializer
+    @property
+    def registered_user_id(self):
```

Comment:
Traditionally we handle that in the view, but that, also, is unsatisfying for different reasons. This, at least, keeps us from having to iterate over the entire list of results each time.